metricbeat/beater/reload.go

```diff
@@ -0,0 +1,223 @@
+package beater
+
+import (
+	"expvar"
+	"fmt"
+	"sync"
+	"time"
+
```

Comment:
About Reload: Yeah, tried different versions, but didn't work. Extracting it and having a separate unpack should work (in a later PR). Not sure why I can't comment above -> Github fail.

Will remove the extra line when rebasing.